data/yam/agama/auto/sle_lvm_encrypted.jsonnet

```diff
@@ -0,0 +1,55 @@
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+    registrationCode: '{{SCC_REGCODE}}'
```

Comment:
we should test it in production job group without registration and using agama.install_url.